src/c++/perf_analyzer/mock_infer_data_manager.h

```diff
@@ -26,18 +26,20 @@
 #pragma once
 
 #include "infer_data_manager.h"
+#include "infer_data_manager_shm.h"
```

Comment:
Including both types of `infer_data_manager` children doesn't create a diamond problem?

Reply:
It would if a single class pulled from both, which doesn't happen.

Reply:
thanks for verifying